plugins/com.google.cloud.tools.eclipse.appengine.libraries.ui/src/com/google/cloud/tools/eclipse/appengine/libraries/ui/CloudLibrariesPage.java

```diff
@@ -101,23 +106,33 @@ public void initialize(IJavaProject project, IClasspathEntry[] currentEntries) {
     if (libraries == null || libraries.isEmpty()) {
       return null;
     }
-  
+
     try {
       if (MavenUtils.hasMavenNature(project.getProject())) {
         BuildPath.addMavenLibraries(project.getProject(), libraries, new NullProgressMonitor());
         return new IClasspathEntry[0];
       } else {
+        Set<LibraryFile> masterFiles = new HashSet<>();
         for (Library library : libraries) {
           if (!library.isResolved()) {
             library.resolveDependencies();
           }
+          masterFiles.addAll(library.getLibraryFiles());
         }
+
+        Library masterLibrary = CloudLibraries.getMasterLibrary();
```

Comment:
I think renaming the local var to `masterContainer` or `masterLibraryContainer` improves readability.